scripts/simapp/start.sh

```diff
@@ -4,7 +4,7 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 
 # Please keep this in sync with the Ports overview in HACKING.md
 TENDERMINT_PORT_GUEST="26657"
-TENDERMINT_PORT_HOST="26657"
+TENDERMINT_PORT_HOST="26658"
```

Comment:
Can you update the ports list in HACKING.md accordingly?

Reply:
I thought I did! What did I leave out?

Reply:
Oh, just didn't push the latest change.